widgets/common/src/deploymentsView/detailsPane/drilldownButtons/common.ts

```diff
@@ -1,6 +1,10 @@
 import { i18nDrillDownPrefix } from '../../common';
+import { SubdeploymentsResult } from './subdeployments-result';
 
 const i18nDrillDownButtonsPrefix = `${i18nDrillDownPrefix}.buttons`;
 
-// eslint-disable-next-line import/prefer-default-export
 export const tDrillDownButtons = Stage.Utils.getT(i18nDrillDownButtonsPrefix);
+
+export const shouldDisplaySubdeploymentButton = (buttonsData: SubdeploymentsResult) => {
+    return !buttonsData.loading && buttonsData.count > 0;
```

Comment:
Optional:

Maybe in that case it would be OK to simply name the variable `result`?
```suggestion
export const shouldDisplaySubdeploymentButton = (result: SubdeploymentsResult) => {
    return !result.loading && result.count > 0;
```

Reply:
I thought about it, but my reasoning was that the context of the word `result` may not be known for the programmer, and that's why I've added word which was more related to the button itself ... but I forgot about the typing, which gives full context to the programmer 😁

So, thanks for a nice catch!
I'll add that, along with the incoming changes

Reply:
Yes, I think the type name is making the change here, so it is self explanatory 🙂 Thanks for addressing this comment, I'm resolving the discussion.